docs/configure/automation/integrations.md

```diff
@@ -71,6 +71,109 @@ if (ActionItem.ResourceNamespace === "api") {
 
 If the Action Item associated with the ticket is marked as `Resolved` or `Fixed`, the third party ticket will automatically close.
 
+### Customizable fields:
+Insights provides ability to customize how the ticket is created at the target ticketing provider in some extent, by adding an a 4th parameter to `createTicket` function.
```

Comment:
update to "Insights supports the ability to customize how tickets are created within a target ticketing provider by adding a 4th parameter to the 'createTicket' function.

Reply:
done